install_config/aggregate_logging.adoc

```diff
@@ -3299,3 +3299,107 @@ link:https://github.com/Scribery/tlog[Tlog].
 |Scrubbed invalid output characters as bytes.
 |===
 endif::openshift-origin[]
+
+== Manual Elasticsearch rollouts
+
+The latest release of the Aggregated Logging stack has updated the Elasticsearch
+Deployment Config object so that it no longer has a Config Change Trigger meaning
+any changes to the DC will not result in an automatic rollout. This was to prevent
+unintended restarts happening in the ES cluster which could create lots of shard
+balancing as cluster members restart.
+
+[[elasticsearch-rolling-restart]]
+=== Performing an Elasticsearch rolling cluster restart
+
+A rolling restart is appropriate to be performed when not changing major versions
```

Comment:
Perhaps consider a few edits here?

----

A rolling restart is appropriate, and recommended, when any of the following changes are made:

 * nodes on which Elasticsearch pods run require a reboot
 * logging-elasticsearch configmap
 * logging-es-* deployment configuration
 * new image deployment, or upgrade

This will be the recommended restart policy going forward.